test/e2e/network/networking_perf.go

```diff
@@ -162,6 +318,10 @@ func networkingIPerfTest(isIPv6 bool) {
 	})
 }
 
+var _ = SIGDescribe("Networking IPerf3 IPv4 [Experimental] [Feature:Networking-IPv4] [Slow] [Feature:Networking-Performance]", func() {
```

Comment:
can we delete or replace one of the other tests here or do you think 
ite better to add this as new?  

Maybe add a //TODO above about how if this proves useful to delete the older tests as well with it.

Reply:
👍